docs/messaging/messaging-spans.md

```diff
@@ -471,15 +470,15 @@ Attributes in the `messaging.message` or `messaging.{system}.message` namespace
 
 For batch operations, per-message attributes are usually different and cannot be set on the corresponding span. In such cases the attributes SHOULD be set on links. See [Batch receiving](#batch-receiving) for more information on correlation using links.
 
-Some messaging systems (e.g., Kafka, Azure EventGrid) allow publishing a single batch of messages to different topics. In such cases, the attributes in `messaging.destination` MAY be
+Some messaging systems (e.g., Kafka, Azure Event Grid) allow publishing a single batch of messages to different topics. In such cases, the attributes in `messaging.destination` MAY be
 set on links. Instrumentations MAY set destination attributes on the span if all messages in the batch share the same destination.
 
 ### Attributes specific to certain messaging systems
 
 All attributes that are specific for a messaging system SHOULD be populated in `messaging.{system}` namespace. Attributes that describe a message, a destination, a consumer, or a batch of messages SHOULD be populated under the corresponding namespace:
 
 * `messaging.{system}.message.*`: Describes attributes for individual messages
-* `messaging.{system}.destination.*`: Describes the destination a message (or a batch) are published to and received from respectively. The combination of attributes in these namespaces should uniquely identify the entity and include properties significant for this messaging system. For example, Kafka instrumentations should include partition identifier.
```

Comment:
I don't think this comment still applies - we should have everything uniquely identifying entity in the general `messaging.destination` namespace.

We also have a note on the `messaging.destination.name` which says "Destination name SHOULD uniquely identify a specific queue, topic or other entity within the broker."